src/ResourceManager/Consumption/Commands.Consumption/Models/PSUsageDetail.cs

```diff
@@ -12,35 +12,44 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Commands.Consumption.Common;
 using System;
 using System.Collections.Generic;
+using Microsoft.Azure.Commands.Consumption.Common;
 using ApiUsageDetail = Microsoft.Azure.Management.Consumption.Models.UsageDetail;
 using MeterDetails = Microsoft.Azure.Management.Consumption.Models.MeterDetails;
 
 namespace Microsoft.Azure.Commands.Consumption.Models
 {
     public class PSUsageDetail
     {
-        public string Id { get; private set; }
-        public string Name { get; private set; }
-        public string Type { get; private set; }
-        public IDictionary<string, string> Tags { get; private set; }
-        public DateTime? UsageStart { get; private set; }
-        public DateTime? UsageEnd { get; private set; }
+        public string AccountName { get; set; }
+        public string AdditionalProperties { get; set; }
```

Comment:
Looks like this changed type - please change back.

Reply:
@maddieclayton @ms-premp As we need this output as string, we keep AdditionalProperties as dictionary type (will depreciate it and label as future breaking change), and introduce a new output parameter as AdditionalInfo with string type.

Reply:
Sounds good - please do the same for Tags/Tag in the output.